src/index.js

```diff
@@ -292,6 +298,6 @@ export function fromTag(tag) {
     case 'FPS_6000':
       return RATE_60;
     default:
-      throw `Unknow Frame Rate ${tag}`;
```

Comment:
This was bothering me so I changed it here, if this doesn't merge, I'll break it out into a separate commit.